src/dispatch.jl

```diff
@@ -475,19 +475,54 @@ Base.:*(A::AbstractArray, α::AbstractMutable) = A .* α
 # Needed for Julia v1.0, otherwise, `broadcast(*, α, A)` gives a `Array` and
 # not a `Symmetric`.
 
-_mult_upper(α, A) = parent(α * LinearAlgebra.UpperTriangular(parent(A)))
-_mult_lower(α, A) = parent(α * LinearAlgebra.LowerTriangular(parent(A)))
+function _mult_triangle(
+    ::Type{T},
+    x,
+    A::T,
+) where {T<:Union{LinearAlgebra.Symmetric,LinearAlgebra.Hermitian}}
+    c = LinearAlgebra.sym_uplo(A.uplo)
+    B = if c == :U
+        parent(x * LinearAlgebra.UpperTriangular(parent(A)))
+    else
+        parent(x * LinearAlgebra.LowerTriangular(parent(A)))
+    end
+    # Intermediate conversion to `Matrix` is needed to work around
+    # https://github.com/JuliaLang/julia/issues/52895
+    return T(Matrix(LinearAlgebra.Symmetric(B, c)), c)
```

Comment:
Didn't you mean
```suggestion
    return T(Matrix(T(B, c)), c)
```
?

Reply:
I don't think it really matters? I guess `T` doesn't hurt.